auth0/resource_auth0_prompt_custom_text.go

```diff
@@ -14,48 +14,55 @@ import (
 	"gopkg.in/auth0.v5/management"
 )
 
-func newPromptCustomText() *schema.Resource {
+var (
+	availablePrompts = []string{
```

Comment:
Created these vars for holding all the prompts and countries so it's a bit more explicit what they are.

Reply:
Nice idea!